backend/queries/volunteers.js

```diff
@@ -0,0 +1,57 @@
+/*
+ANIME BENSALEM, BRIAHANA MAUGÉ, JOSEPH P. PASAOA
+FELLOWS Route Queries | Capstone App (Pursuit Volunteer Mgr)
+*/
+
+/* DB CONNECTION */
+const db = require('../db/db');
+
+
+/* QUERIES */
+
+// Get all volunteers
+const getAllVolunteers = async () => {
+    const selectQuery = `
+    SELECT *
+    FROM volunteers
+    ORDER BY v_first_name ASC
+  `;
+    return await db.any(selectQuery);
+}
+
+// Get all new (unconfirmed) volunteers
+const getNewVolunteers = async () => {
+    const selectQuery = `
+      SELECT *
+      FROM volunteers
+      WHERE confirmed = FALSE
+      ORDER BY v_first_name ASC
+    `;
+    return await db.any(selectQuery);
+}
+
+// Get volunteer by email 
+const getVolunteerByEmail = async (vEmail) => {
+  const selectQuery = `
+  SELECT *
+  FROM volunteers
+  WHERE v_email = $/vEmail/
+  `;
+  return await db.any(selectQuery, {vEmail});
```

Comment:
I think it's better to always use a db.one when we're expecting one result, this will allow us avoid an extra check for the length when processing the data. if something goes wrong here (no match) it will be caught by the try/catch

Reply:
You're right, I fixed it on my copy but I haven't pushed it yet. this was as we were starting class